x-pack/test/security_solution_api_integration/config/serverless/config.base.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrConfigProviderContext } from '@kbn/test';
+
+export interface CreateTestConfigOptions {
+  testFiles: string[];
+  junit: { reportName: string };
+}
+
+export function createTestConfig(options: CreateTestConfigOptions) {
+  return async ({ readConfigFile }: FtrConfigProviderContext) => {
+    const svlSharedConfig = await readConfigFile(
+      require.resolve('../../../../test_serverless/shared/config.base.ts')
+    );
+
+    return {
+      ...svlSharedConfig.getAll(),
+      kbnTestServer: {
+        ...svlSharedConfig.get('kbnTestServer'),
+        serverArgs: [...svlSharedConfig.get('kbnTestServer.serverArgs'), '--serverless=security'],
+      },
```

Comment:
`x-pack/test_serverless/functional/test_suites/security/config.ts` has a `serverlessProject: 'security',` property. Is it a proper way to configure this?

Reply:
The `serverlessProject` parameter can take on the values 'es', 'oblt', or 'security' to indicate the specific project it pertains to. Given that x-pack/test_serverless encompasses all three solutions, the project should clearly identify itself as one of them. However, in our new folder, where we exclusively work with the 'security' project, we can directly pass it to the `serverArgs`, as this `serverlessProject` variable gets compiled accordingly.